lib/configs/app_constans.dart

```diff
@@ -27,6 +27,7 @@ class AppConstants {
   static String tripsSearchToDate = 'To date';
 
   static String emailInputLabel = 'Enter email';
+  static String loginInputLabel = 'Enter email or username';
```

Comment:
Change label form 'Enter email' to 'mail@example.com' and 'mail@example.com or username'

Reply:
Placeholder and label are different things, which one I need to change?

Reply:
Moved to #46